src/pages/prescreen-1a.tsx

```diff
@@ -50,7 +50,7 @@ class Prescreen1a extends React.Component<Prescreen1aProps, Prescreen1aState> {
 
   async handleDateChange(name, value) {
     const {userStateActions} = this.props
-    if (name === "birthDatePicked") {
+    if (value && name === "birthDatePicked") {
```

Comment:
If `value` is not defined, then we should `setBirthDate(null)` and `setRetireDate(null)` so that it properly clears out the field!

Reply:
I see. That makes sense.

Reply:
Hi Kevin, can you please make that change so we can merge this.

Reply:
![Screen Shot 2020-04-12 at 10 51 34 AM](https://user-images.githubusercontent.com/50506809/79071795-9e946a80-7cab-11ea-89f2-64795a3be5cc.png)

I tried to change `setBirthDate(null)` and `setRetireDate(null)` but the crash persisted.

The problem seems to be that `setBirthDate()` does not take 'null' as it's parameters. It only accepts type "Date"

Reply:
Is there a specific null value for Date in this case?

Reply:
> The problem seems to be that setBirthDate() does not take 'null' as it's parameters. It only accepts type "Date"

Yep, see the comment I made [here!](https://github.com/codeforboston/windfall-elimination/issues/181#issuecomment-611187514) Basically, we need to update the `UserStateActions` interface to accept `null` as an argument, like so:
```
setBirthDate: (date: Date | null) => void
setRetireDate: (date: Date | null) => void
```

Then, you'll probably get an error in `user-state-manager.tsx`, which you'll likely have to update as follows:
```
setBirthDate: date => setBirthDate(date ? startOfDay(date) : null),
setRetireDate: date => setRetireDate(date ? startOfDay(date) : null)
```

Reply:
That worked. Thank you very much. 
![Screen Shot 2020-04-12 at 4 19 30 PM](https://user-images.githubusercontent.com/50506809/79078877-6efc5700-7cd9-11ea-81e8-0350371b6be9.png)
Now when the date is deleted, there is an annoying NaN below the DatePicker. I will try to get rid of it and then commit the change.

Reply:
Hm strange, because the `DatePicker` component should allow `null` as a value. Maybe try setting the value to `undefined` if `birthDate === null`? Let me know what you find out!

Reply:
It seems that dayjs(dateOfBirth) will turn null into Invalid Date. 
I fixed it by adding a condition check. And the NaN is gone.
![Screen Shot 2020-04-12 at 5 01 01 PM](https://user-images.githubusercontent.com/50506809/79079789-3790a900-7cdf-11ea-8e11-500e43baba50.png)

That's the easiest way I can think of. But let me know if there is a better fix.  :)

Reply:
Also, I'm really not familiar with Github pull requests. 
I've committed a new change. But is there anything else I need to do? Should I create another pull request?

Reply:
Ah, that makes sense. I think a simpler solution would be to avoid updating `this.setRetireDate` at all and simply call `userStateActions.setRetireDate(null)` directly on [line 64](https://github.com/codeforboston/windfall-elimination/pull/182/files#diff-a2ec762a64d429851a563ab208373d53R64). What do you think?

Reply:
The less code, the better. That makes the code much cleaner.

Do I need to do anything other than commit to finish this pull request?

Reply:
Nope, I think you're all set! Once we can confirm everything works as expected we'll go ahead and approve and merge your PR.

Reply:
Awesome. Thanks for the help. Much appreciated.